Assets/Scripts/pipe.cs

```diff
@@ -9,12 +10,6 @@ public class pipe : MonoBehaviour
     private gridManager gridControl;
     [SerializeField] private Vector2[] allowedDirections;
 
-    enum DIRECTION {
```

Comment:
Agree with the decision to rather have the connection coordinates saved on each prefab instead of ENUM stuff.